package.json

```diff
@@ -78,6 +78,7 @@
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
     "react-helmet-async": "^1.0.4",
+    "react-router-dom": "^5.2.0",
```

Comment:
Do we need this here so that it works in the docs viewer?

Reply:
yes it is